.github/workflows/pull_request.yml

```diff
@@ -38,47 +38,84 @@ jobs:
           branch_name='preview-${{ github.event.pull_request.number }}'
           echo branch doesnt exist
           git checkout -b $branch_name || echo branch exists
-          git push --force --set-upstream origin $branch_name
+          git push --set-upstream origin $branch_name || echo branch exists remotely
         shell: bash
 
+
     outputs:
-      public_repository: "${{ steps.public_repository.outputs.public }}"
+      toggle_website: "${{ env.RENDER_WEBSITE }}"
       toggle_spell_check: "${{ env.SPELL_CHECK }}"
+      spell_error_min: "${{ env.SPELL_ERROR_MIN }}"
+      toggle_style_code: "${{ env.STYLE_CODE }}"
       toggle_url_check: "${{ env.URL_CHECKER }}"
+      url_error_min: "${{ env.URL_ERROR_MIN }}"
+      toggle_quiz_check: "${{ env.CHECK_QUIZZES }}"
+      quiz_error_min: "${{ env.QUIZ_ERROR_MIN }}"
+      toggle_md_linter: "${{ env.MARKDOWN_LINTER }}"
+      toggle_readability: "${{ env.READABILITY_REPORT }}"
       toggle_render_preview: "${{ env.RENDER_PREVIEW }}"
+      toggle_dictionary_sort: "${{ env.DICTIONARY_SORT }}"
       rendering_docker_image: "${{ env.RENDERING_DOCKER_IMAGE }}"
-      toggle_readability: "${{ env.READABILITY_REPORT }}"
 
 ########################## Make the error reports ##############################
-  spell-check:
-    name: Check spelling
+  ottr-reports:
+    name: Run Reports
     needs: yaml-check
-    if: ${{needs.yaml-check.outputs.toggle_spell_check == 'yes'}}
-    uses: ottrproject/ottr-reports/.github/workflows/report-maker.yml@main
-    with:
-      check_type: spelling
-      error_min: 3
-    secrets:
-      gh_pat: ${{ secrets.GH_PAT }}
-
-  url-check:
-    name: Check URLs
+    if: ${{needs.yaml-check.outputs.toggle_spell_check == 'true' || needs.yaml-check.outputs.toggle_url_check == 'true' || needs.yaml-check.outputs.toggle_quiz_check == 'true'}}
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: write
+
+    steps:
+    - name: Checkout Actions Repository
+      uses: actions/checkout@v4
+      with:
+        fetch-depth: 0
+
+    - name: Run Reports
+      id: run-reports
+      uses: ottrproject/ottr-reports@main
+      with:
+        check_spelling: ${{needs.yaml-check.outputs.toggle_spell_check}}
+        spelling_error_min: ${{needs.yaml-check.outputs.spell_error_min}}
+        check_urls: ${{needs.yaml-check.outputs.toggle_url_check}}
+        url_error_min: ${{needs.yaml-check.outputs.url_error_min}}
+        check_quiz_form: ${{needs.yaml-check.outputs.toggle_spell_check}}
```

Comment:
I think this needs to be fixed upstream in the template, but it's using the `toggle_spell_check` value for `check_quiz_form`

Reply:
yes sorry I fixed this with the main template but forgot elsewhere - I think we might need another sync

Reply:
https://github.com/ottrproject/OTTR_Template/pull/15